test/integration/submission/submission-schema-migration.spec.ts

```diff
@@ -371,11 +373,20 @@ describe('schema migration api', () => {
       assertNoMigrationErrors(res, VERSION);
 
       migration.invalidDonorsErrors.length.should.equal(1);
-      const errorObj = migration.invalidDonorsErrors[0].errors[0].primary_diagnosis[0];
+
+      const tumourStagingError = migration.invalidDonorsErrors[0].errors[0].primary_diagnosis[0];
       chai
-        .expect(errorObj)
+        .expect(tumourStagingError)
         .to.have.property('errorType', SchemaValidationErrorTypes.INVALID_ENUM_VALUE);
-      chai.expect(errorObj).to.have.property('fieldName', TUMOUR_STAGING_SYSTEM);
+      chai.expect(tumourStagingError).to.have.property('fieldName', TUMOUR_STAGING_SYSTEM);
+
+      const presentingSymptomError =
+        migration.invalidDonorsErrors[0].errors[0].primary_diagnosis[1];
+      chai.expect(presentingSymptomError).to.deep.include({
+        errorType: SchemaValidationErrorTypes.INVALID_ENUM_VALUE,
+        fieldName: PRESENTING_SYMPTOMS,
+        info: { value: ['Nausea'] },
```

Comment:
I updated this test to remove a code-list value from `presenting_symptoms` so the record becomes invalid to make sure the re-validate is working. Looks like the toString is working since `typeof Array` is object